vizro-core/src/vizro/static/css/typography.css

```diff
@@ -50,17 +44,18 @@ h4,
 }
 
 .body-ui-link-01 {
-  color: var(--text-secondary);
-  font-size: var(--text-size-03);
   font-weight: var(--text-weight-semibold);
-  letter-spacing: var(--letter-spacing-body-ui-01);
-  line-height: var(--text-size-04);
   text-decoration: underline;
 }
 
 p,
 label,
-.body-ui-02 {
+h5,
+h6,
+ul,
+li,
+.body-ui-02,
+.body-ui-link-02 {
```

Comment:
Just to understand as an example, what happens if we don't have this custom css here? Does it look really awful or just a little off?

Reply:
Depends what you consider as awful 😄 If you have some bullet points and some paragraph text in the card-text for example, you would see that the text is not consistently formatted within the card if this code is not in here. So I would consider it already as necessary for visual consistency. But it would not look completely off.